hudi-common/src/main/java/org/apache/hudi/common/model/DefaultHoodieRecordPayload.java

```diff
@@ -37,6 +39,9 @@
  */
 public class DefaultHoodieRecordPayload extends OverwriteWithLatestAvroPayload {
 
+  public static final String METADATA_EVENT_TIME_KEY = "metadata.event_time.key";
```

Comment:
should we also support this at the  `OverwriteWithLatestAvroPayload` level

Reply:
yea can be a good idea